### PR TITLE
Remove web/package.json, move scripts to package.json

### PR DESCRIPTION
**Pull request by ravicious**
I'm trying to make a new entry point for teleterm.e and the duplication makes adding new scripts much harder. This PR removes `web/package.json` so that we can refer to our packages in the root package.json again.

Removed scripts:

Leftovers from webapps Dockerfile and CI on GCB:
- package-term-ci
- build-and-package-term-ci
- build-and-package-term

Leftovers from webapps Makefile, presumably:
- build-oss
- build-e

Leftovers after webapps merge; use `build-ui*` variants instead.
- build-teleport
- build-teleport-oss
- build-teleport-e

### CODE REVIEW

**Review by gzdunek — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
     "build-ui-oss": "yarn workspace @gravitational/teleport build --output-path=../../../webassets/teleport/app",
     "build-ui-e": "yarn workspace @gravitational/teleport.e build --output-path=../../../webassets/e/teleport/app",
-    "build-teleport": "yarn workspace @gravitational/namespaces build-teleport",
-    "start-teleport": "yarn workspace @gravitational/namespaces start-teleport",
-    "build-term": "yarn workspace @gravitational/namespaces build-term",
-    "start-term": "yarn workspace @gravitational/namespaces start-term",
-    "package-term": "yarn workspace @gravitational/namespaces package-term",
-    "build-native-deps-for-term": "yarn workspace @gravitational/namespaces build-native-deps-for-term",
-    "build-and-package-term-ci": "yarn workspace @gravitational/namespaces build-and-package-term-ci",
+    "build-teleport": "yarn build-teleport-oss && yarn build-teleport-e",
```
Comment (ryanclark) [inline package.json:8]: Should this call `build-ui` and then `build-teleport-e` and `build-teleport-oss` can go?

Comment (hatched) [reply package.json:8]: yes plz :) This was my intention with the `build-ui`, `build-ui-oss`, and `build-ui-e` naming scheme.

File: package.json [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
+    "start-term": "yarn workspace @gravitational/teleterm start",
+    "package-term": "yarn workspace @gravitational/teleterm package",
+    "build-native-deps-for-term": "yarn workspace @gravitational/teleterm build-native-deps",
     "storybook": "start-storybook -p 9002 -c web/.storybook -s web/.storybook/public",
     "test": "jest",
+    "test-coverage": "jest --coverage && web/scripts/print-coverage-link.sh",
+    "test-update-snapshot": "yarn test -- --updateSnapshot",
+    "tdd": "jest --watch",
     "lint": "yarn prettier-check && yarn eslint",
     "eslint": "eslint --quiet --ext .js,.jsx,.ts,.tsx web/ e/",
```
Comment (hatched) [inline package.json:23]: mind sorting alphabetically?

File: web/package.json [hunk 3]
```diff
@@ -1,43 +0,0 @@
...
-    "build-and-package-term-ci": "yarn build-term && yarn package-term-ci",
-    "start-term": "yarn workspace @gravitational/teleterm start",
-    "start-teleport": "yarn workspace @gravitational/teleport start",
-    "start-teleport-e": "yarn workspace @gravitational/teleport.e start",
-    "build-teleport": "yarn build-teleport-oss && yarn build-teleport-e",
-    "build-teleport-oss": "yarn workspace @gravitational/teleport build --output-path=../../../webassets/teleport/app",
-    "build-teleport-e": "yarn workspace @gravitational/teleport.e build --output-path=../../../webassets/e/teleport/app",
-    "build-oss": "yarn build-teleport-oss",
-    "build-e": "yarn build-teleport-e",
-    "nop": "exit 0"
```
Comment (hatched) [inline web/package.json:25]: @ryanclark `development` uses the `nop` target as the default `NPM_SCRIPT`. Just a note that we'll want to remove it from there...

**Review by hatched — APPROVED** [review]

**Review by avatus — APPROVED** [review]

File: web/package.json (hunk 3 above)
Comment (ravicious) [reply web/package.json:25]: I'll leave it then in the root package.json. AFAIK there's no easy way around it when using a Dockerfile.

File: package.json (hunk 1 above)
Comment (ravicious) [reply package.json:8]: Should we just remove all three `build-teleport*` scripts completely then? I don't see them being referenced anywhere, other than `e/web/README.md`.

Comment (hatched) [reply package.json:8]: Yeah sure that sounds good

**Review by ryanclark — APPROVED** [review]

**Pull request merged**